README.md

```diff
@@ -848,6 +850,91 @@ Similarly, you can get `Arbitrary` and `Cogen` instances for every `ValueEnum` s
 import enumeratum.values.scalacheck._
 ```
 
+## Quill
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.beachape/enumeratum-quill_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.beachape/enumeratum-quill_2.11)
+
+### SBT
+
+To use enumeratum with [Quill](http://getquill.io):
+
+```scala
+libraryDependencies ++= Seq(
+    "com.beachape" %% "enumeratum-quill" % enumeratumQuillVersion
+)
+```
+
+To use with ScalaJS:
+
+```scala
+libraryDependencies ++= Seq(
+    "com.beachape" %%% "enumeratum-quill" % enumeratumQuillVersion
+)
+```
+
+### Usage
+
+#### Enum
+
+```scala
+import enumeratum._
+
+sealed trait ShirtSize extends EnumEntry
+
+case object ShirtSize extends Enum[ShirtSize] with QuillEnum[ShirtSize] {
+
+  case object Small  extends ShirtSize
+  case object Medium extends ShirtSize
+  case object Large  extends ShirtSize
+
+  val values = findValues
+
+}
+
+case class Shirt(size: ShirtSize)
+
+import io.getquill._
+
+lazy val ctx = new PostgresJdbcContext(SnakeCase, "ctx")
+import ctx._
+
+ctx.run(query[Shirt].insert(_.size -> lift(ShirtSize.Small: ShirtSize)))
+
+ctx.run(query[Shirt]).foreach(println)
+```
+- Note that an explicit cast to the `EnumEntry` trait (eg. `ShirtSize.Small: ShirtSize`) is required when binding hardcoded `EnumEntry`s
+
+#### ValueEnum
+
+```scala
+import enumeratum._
+
+sealed abstract class ShirtSize(val value: Int) extends IntEnumEntry
+
+case object ShirtSize extends IntEnum[ShirtSize] with IntQuillEnum[ShirtSize] {
+
+  case object Small  extends ShirtSize(1)
+  case object Medium extends ShirtSize(2)
+  case object Large  extends ShirtSize(3)
+
+  val values = findValues
+
+}
+
+case class Shirt(size: ShirtSize)
+
+import io.getquill._
+
+lazy val ctx = new PostgresJdbcContext(SnakeCase, "ctx")
+import ctx._
+
+ctx.run(query[Shirt].insert(_.size -> lift(ShirtSize.Small: ShirtSize)))
+
+ctx.run(query[Shirt]).foreach(println)
+```
+- Note that an explicit cast to the `ValueEnumEntry` abstract class (eg. `ShirtSize.Small: ShirtSize`) is required when binding hardcoded `ValueEnumEntry`s
```

Comment:
No big deal; but this isn't an explicit cast; just an explicit type annotation :)

Reply:
Ah, I knew that it wasn't an explicit cast (`asInstanceOf`), but I forgot what it was called, thanks

Reply:
Actually, if we were to be pedantic, I guess this should be called a [Type Ascription](https://docs.scala-lang.org/style/types.html#ascription) instead of a Type Annotation?

Reply:
Also, you missed out on one of the "explicit cast"s. I've opened a PR (#172) to fix both.